### DISCUSSION TITLE
Data Assimilation in Observation Space

### DISCUSSION THREAD

**Discussion by cbblanke** [NASA-LIS/LISF#1157] [Q&A] [answered]
The LIS 7.2 notes say "Data assimilation is performed on the observation grid".  I believe this means the observation may contain multiple grid cells.  Is this documented in more detail somewhere? Where can I find an example or testcase that shows how to do this? Does the observation dataset have to be on a grid of some type (as opposed to point data, satellite swaths, etc.)?  

Mathematically, does it adjust all the grid cells collectively to try to match the observation on average (as opposed to just applying the same observation value to all the grid cells in the footprint)?  

Thanks,
Clay

**Comment by dmocko** [comment]
Hi @cbblanke 

As a non-DA expert, I don't know the answers to these questions.  I will ask the DA experts on the team to address this issue.

I will also ask them to prepare some slides for the next LIS tag-up meeting, so we can discuss in further detail.  After that, we can also share the slides here for other users.

**Comment by dmocko** [comment]
I just noticed the "LIS 7.2" portion of your question, so not sure if we'll be able to provide a detailed answer about older code.

The public LISF-7.4 was just released, so we strongly recommend that you upgrade, and we answer questions about current code.

**Comment by cbblanke** [comment]
I have the same question about 7.4, it's just that this feature appears in the change log for 7.2 and that's the only place I see it mentioned.

**Comment by LIS-navari** [comment, marked as answer]
Hi Clay, 

LIS can perform the DA on the observation grid. 
Please see the attached figure. 
I believe you can run your test case in the observation space. You need to change one entry in the LIS config file. The rest are like DA on model space. Please note that I have not used this feature. 

For the following entry 
"Data assimilation observation domain file:"
you have to use the domain file generated in the process of generating the observation CDF. 
For DA in model space, we use the same entry we use for
“LIS domain and parameter data file”

I hope this helps. 
However, you can bring up this question in our next tag-up meeting. 

![Screen Shot 2022-07-25 at 1 36 23 PM](https://user-images.githubusercontent.com/43690895/180839641-320cda61-f91d-469b-a67b-a07e4b78c95d.png)
